scripts/processing_server.py

```diff
@@ -0,0 +1,98 @@
+import os
+import xeauth
+import pymongo
+import rframe
+import requests
+from fastapi import FastAPI, Request, Depends, status, HTTPException
+from fastapi.security import HTTPBearer
+from typing import Any, Optional, Union, List 
+
+from utilix.processing_requests import ProcessingRequest, ProcessingJob
+
+schemas = {
+    'processing_request': ProcessingRequest,
+    'processing_job': ProcessingJob,
+}
+
+mongo_user = os.getenv('MONGO_USER')
+mongo_pass = os.getenv('MONGO_PASS')
+
+url = f'mongodb://{mongo_user}:{mongo_pass}@xenon1t-daq.lngs.infn.it:27017/cmt2'
```

Comment:
Similar here, this is replicating a lot of code that's already in utilix. Can we just use `xent_collection` [here](https://github.com/XENONnT/utilix/blob/master/utilix/rundb.py#L579)?

Reply:
Again, this is just a demo api server which will run on an lngs machine so should connect directly to the daq replica

Reply:
Yeah so we can specify that in the config for that machine, at least that would be my preference

Reply:
done.